obj2ms3dascii.mjs

```diff
@@ -53,6 +53,8 @@ export function obj2ms3dascii( obj, mtl ) {
       currentObject.usemtl = li[ 1 ];
     }
   } );
+  
+  const objects = unfiltered_objects.filter((x) => x.f.length > 0);
```

Comment:
[SHOULD] I think we should emit a warning when we got objects or groups without faces

Reply:
ah since `g` can exist inside of `o` and in that case the `o` can have zero faces because we treat `o` and `g` in the same way I see.
I'd prefer handle groups in more appropriate way since the name of object still matters right

Reply:
it's actually okay to get an object/group without faces -- the way it curretly works is it either:

In the case of no groups:
- Encounters "o" New Object tag, creating the new object and pushing it into the list
- Encounters faces, appending them to the object's face list

In the case of groups:
- Encounters the "o" New Object tag, creating the new object and pushing it into the list
- Encounters the "g" New Group tag, creating the new group and pushing it into the list (the "parent" object never got any faces! But that's okay, since the object itself doesn't contain faces, it contains groups of faces)
- Encounters faces, appending them to the group's face list
- Filters out empty objects (in this case, the "o" tag which has served it's purpose and never got faces added)

Reply:
could I see a sample model?

Reply:
Here's a sample model I've used successfully with my changes
[box_mesh.zip](https://github.com/0b5vr/obj2ms3dascii/files/8105548/box_mesh.zip)

Reply:
Yes the structure is what I have expected. I'd prefer having a name in format of like `${ object.name }/${ group.name }` in these situations ("Cube/Cube_Cube_Black" in this example for example).
I can do this by myself though.

Reply:
I agree, sadly I simply do not have any more time to modify the file myself. Thank you for your work :)

Reply:
gotcha, I'll merge this as is, I'll do this in a separated PR.